dietpi/dietpi-software

```diff
@@ -11165,6 +11350,30 @@ _EOF_
 			ln -sf "$FP_DIETPI_USERDATA_DIRECTORY"/steam "$HOME"/.steam
 
 		fi
+		
+		#------------------ Home Automation: Home Assistant ------------------
+		INSTALLING_INDEX=157
+		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
+
+			cat << _EOF_ > /etc/systemd/system/home-assistant.service
+[Unit]
+Description=Home Assistant
+After=network.target
+
+[Service]
+Type=simple
+User=%i
+ExecStart=/srv/homeassistant/bin/hass -c "/home/homeassistant/.homeassistant"
+
+[Install]
+WantedBy=multi-user.target
+
+_EOF_
+
+			# TODO: Validate what happens when user data moves, need to find a SD card. 
+			#mkdir -p "$FP_DIETPI_USERDATA_DIRECTORY"/homeassistant
+			#ln -sf "$FP_DIETPI_USERDATA_DIRECTORY"/homeassistant /home/homeassistant/.homeassistant
```

Comment:
must have missed that as I thought I already added it. Whoops! I have updated the apply section now.